src/Backend/Modules/MediaLibrary/Ajax/MediaItemGetAllById.php

```diff
@@ -34,6 +34,12 @@ protected function getMediaItems(): array
             return [];
         }
 
-        return $this->get('media_library.repository.item')->findById($ids);
+        $mediaItems = [];
```

Comment:
why not just sort them in the method itself?

Reply:
The id's are passed in a given order. This is the best way to sort the images in my opinion.

Reply:
the current implementation will get all items in 1 query, you are running a separate query per item, so that solution is not acceptable.
You can use the initial implementation and then sort it by the order of the ids.

Reply:
Ok, but wouldn't it be better to retrieve the images by the given library instead of all the id's? It would be easier to preserve the order and it makes more sense instead of sending all the id's.